src/js/components/common/dialogs/physicaldeviceonboarding.js

```diff
@@ -46,13 +46,32 @@ export default class PhysicalDeviceOnboarding extends React.Component {
 
   render() {
     const self = this;
+    var token = (this.props.org || {}).tenant_token;
```

Comment:
this should be just `self.props.token`, but I'll edit this once this is merged

Reply:
Okey. I am merging the PR and leaving it in your hands for the final working version.